lib/src/config/config.rs

```diff
@@ -222,6 +223,22 @@ impl Default for FileStorageConfig {
     }
 }
 
+impl Debug for FileStorageConfig {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("FileStorageConfig")
+            .field("database_parent", &self.database_parent)
+            .field("peer_key_path", &self.peer_key_path)
+            .field("peer_key", &self.peer_key.as_ref().map(|_| "***"))
+            .field("voting_key_path", &self.voting_key_path)
+            .field("voting_key", &self.voting_key.as_ref().map(|_| "***"))
+            .field("signing_key_path", &self.signing_key_path)
+            .field("signing_key", &self.signing_key.as_ref().map(|_| "***"))
+            .field("fee_key_path", &self.fee_key_path)
+            .field("fee_key", &self.fee_key.as_ref().map(|_| "***"))
+            .finish()
```

Comment:
We need to take in mind the `validator` feature. I think something like this might work:
```suggestion
        let debug_struct = f.debug_struct("FileStorageConfig")
            .field("database_parent", &self.database_parent)
            .field("peer_key_path", &self.peer_key_path)
            .field("peer_key", &self.peer_key.as_ref().map(|_| "***"));
         #[cfg(feature = "validator")]
         {
            debug_struct = debug_struct.field("voting_key_path", &self.voting_key_path)
            .field("voting_key", &self.voting_key.as_ref().map(|_| "***"))
            .field("signing_key_path", &self.signing_key_path)
            .field("signing_key", &self.signing_key.as_ref().map(|_| "***"))
            .field("fee_key_path", &self.fee_key_path)
            .field("fee_key", &self.fee_key.as_ref().map(|_| "***"))
         }
         debug_struct.finish()
```

Reply:
We have just fixed it, thank you